modules/end-user-guide/pages/stack-samples.adoc

```diff
@@ -0,0 +1,12 @@
+:_content-type: REFERENCE
+:description: Stack samples
+:keywords: user-guide, stacks
+:navtitle: Stack samples
+:page-aliases: supported-languages.adoc
+
+[id="stack-samples"]
+= Stack samples
+
+To demonstrate the capabilities of {prod} as remote development environment, {prod} contains stack samples using various programming languages. Each sample includes a devfile and you can use them as a reference to bootstrap a new project. You can customize the samples if you are a {prod-short} administrator.
```

Comment:
Just nitpicks; feel free to discard the suggested version and stick to your own.

xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```
To demonstrate the capabilities of {prod} as a remote development environment, {prod} contains stack samples for various programming languages. 
```
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Assuming "them" means the devfile:
```
Each sample includes a devfile which you can use as a reference to bootstrap a new project. 
```
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```
If you are a {prod-short} administrator, you can customize the samples.
```